nion/swift/model/Graphics.py

```diff
@@ -860,6 +882,13 @@ def used_fill_style(self) -> typing.Optional[str]:
             return "rgba(0, 0, 255, 0.1)"
         return None
 
+    @used_fill_style.setter
```

Comment:
Same as above... `used_fill_style` is a computed value and should not have a setter.